Makefile

```diff
@@ -393,6 +393,125 @@ generate:
 	$(CONTROLLER_GEN_CMD) rbac:roleName=spod paths="./internal/pkg/daemon/..." output:rbac:stdout >> deploy/base/role.yaml
 	$(CONTROLLER_GEN_CMD) rbac:roleName=spo-webhook paths="./internal/pkg/webhooks/..." output:rbac:stdout >> deploy/base/role.yaml
 
+## Bundle packaging begins here
+## read more at https://sdk.operatorframework.io/docs/olm-integration/tutorial-bundle/
+
+.PHONY: operator-sdk
+OPERATOR_SDK = $(BUILD_DIR)/operator-sdk
+operator-sdk: ## Download sdk locally if necessary.
+ifeq (,$(wildcard $(OPERATOR_SDK)))
+ifeq (,$(shell which operator-sdk 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPERATOR_SDK)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.17.0/operator-sdk_$${OS}_$${ARCH} ;\
```

Comment:
The version should be managed in dependencies.yaml

Reply:
Done